manager/docker_manager.go

```diff
@@ -0,0 +1,1183 @@
+// Copyright 2014 Google Inc. All Rights Reserved.
```

Comment:
Im redoing the patch.  I cant seem to rebase it just endless loop even
after i fix all merge errors.  Probably do to not using git mv correctly
first time around.
On Jan 14, 2016 3:59 PM, "Tim St. Clair" <notifications@github.com> wrote:

> In manager/docker_manager.go
> <https://github.com/google/cadvisor/pull/1049#discussion_r49804713>:
>
> > @@ -0,0 +1,1183 @@
> > +// Copyright 2014 Google Inc. All Rights Reserved.
>
> (nit) 2016
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/google/cadvisor/pull/1049/files#r49804713>.
>